common/src/main/java/com/scottlogic/deg/common/profile/Field.java

```diff
@@ -23,12 +23,26 @@
     public final Types type;
     private final boolean unique;
     private final String formatting;
+    private final boolean internal;
 
     public Field(String name, Types type, Boolean unique, String formatting) {
         this.name = name;
         this.type = type;
         this.unique = unique;
         this.formatting = formatting;
+        this.internal = false;
+    }
+
+    public Field(String name, Types type, boolean unique, String formatting, boolean internal) {
```

Comment:
We shouldn't have 2 constructors here. There is only one instance creation in the main java files and most of the tests are using the `FieldBuilder` functions so it shouldn't be onerous to replace the constructor.